bytecomp/matching.ml

```diff
@@ -2824,6 +2825,7 @@ let find_in_pat pred =
         find_rec p || find_rec q
     | Tpat_constant _ | Tpat_var _
     | Tpat_any | Tpat_variant (_,None,_) -> false
+  | Tpat_exception _ -> assert false
```

Comment:
Is there an indentation issue or did it mean something else ?

Reply:
Looks like indentation/copy-paste failure on my part; it's now fixed.

Thanks!